generator/mapper.go

```diff
@@ -123,3 +123,72 @@ func getMappedCatalogControlsFromImport(importedCatalog *catalog.Catalog, profil
 	}
 	return newCatalog, nil
 }
+
+func getCatalogForImport(ctx context.Context, i profile.Import, c chan *catalog.Catalog, e chan error) {
```

Comment:
@anweiss 
This method traverses the entire `import` tree and finds the catalog concurrently. The first goroutine that finds it pushes it to the channel and all other goroutines get killed thanks to the `context` package.
Let me know if this approach works better in stead of the old fashioned `LNR` tree parsing